packages/amplify-cli-core/src/errors/amplify-exception.ts

```diff
@@ -85,6 +85,7 @@ export type AmplifyErrorType =
   | 'DirectoryError'
   | 'DirectoryAlreadyExistsError'
   | 'DuplicateLogicalIdError'
+  | 'EnvironmentConfigurationError'
   | 'EnvironmentNameError'
```

Comment:
Should this be  ```EnvironmentNameFault ```. Since  this check if Cx has provided envName with the  get/import commands ?

Reply:
EnvironmentNameError indicates a user error that has a resolution; we are using Fault to indicate actual errors with the CLI

Reply:
okay , I was thinking it the other way . Thank you